bin/update-data.py

```diff
@@ -97,39 +97,36 @@ def load_ecdc_diff_data(result, dates):
   dt = latest_day - first_day
   if verbosity > 0:
     print('Reading ECDC daily diff data ...')
-  diff_data = None
   try:
     diff_data = pd.read_csv('https://opendata.ecdc.europa.eu/covid19/casedistribution/csv')
     diff_data.drop(['day', 'month', 'year', 'geoId', 'countryterritoryCode', 'popData2019'], axis=1, inplace=True)
     diff_data.to_csv(os.path.join(src_path, 'ecdc-casedistribution.csv'), index=False)
   except urllib.error.URLError as e:
     print(e, file=sys.stdout)
     print('Falling back to locally cached ECDC data ...')
-  if diff_data is None:
     diff_data = pd.read_csv(os.path.join(src_path, 'ecdc-casedistribution.csv'))
-  if diff_data is not None:
```

Comment:
`pd.read_csv()` from a file can also fail, e.g. if the file doesn't exist yet. Thus its necessary to handle this case.

Reply:
But the File is part of the Repository and should never not exist! But also understand this